index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200;300;400;500;700&family=Roboto:wght@100;300;500;700&display=swap">
+</head>
+<body>
+    <div class="head-section">
+        <header>
+         <nav>
+            <a class="logo" href="#">My logo</a>
```

Comment:
- [ ] As requested by the previous reviewer, kindly move `a` tag outside the `nav` tag.